dashtrends.php

```diff
@@ -114,7 +114,11 @@ protected function getData($date_from, $date_to)
         } else {
             $tmp_data['visits'] = AdminStatsController::getVisits(false, $date_from, $date_to, 'day');
             $tmp_data['orders'] = AdminStatsController::getOrders($date_from, $date_to, 'day');
-            $tmp_data['total_paid_tax_excl'] = AdminStatsController::getTotalSales($date_from, $date_to, 'day');
+            if (version_compare(_PS_VERSION_, '1.7.8.0') >= 0) {
```

Comment:
The difference is not obvious, maybe you could add a comment explaining what this last parameter is used for?

Reply:
Fixed